### DISCUSSION TITLE
Duplicate Titles in OA

### DISCUSSION THREAD

**Discussion by IamMaris** [openaudible/openaudible#827] [Q&A] [answered]
I purchased a new computer and downloaded the OA app to it, then moved my previous downloaded books to the proper folders. When I did a full library sync from Audible I ended up with many duplicated book names. One from the files I had moved  (showing it was downloaded), and the other apparently direct from Audible (indicated not downloaded). 

Is there a way to fix this? I tried a second full library sync, and a refresh files, but still have duplicate names. I obviously did something wrong with the way I attempted to transfer my files over, but I don't know what. Any suggestions  for ow to remove the duplicate titles will be very appreciated.

**Comment by openaudible** [comment, marked as answer]
Uh... You may need to start over with a fresh library, then do a full rescan... then you can drag and drop your old AAX files onto the OA app to avoid the download.. But if you get duplicates at that step, you may need to just re-download the files. Hard to know for sure!

**Reply by IamMaris** [reply]
Thank you for responding. When I transferred my books over, it seems some had a readable name, and others had just numbers, and that's where the duplicates came in.  Starting over helped a lot in figuring this out.
